From 62dd2a08e2bf3550221d2885f1f35adc60ab2e24 Mon Sep 17 00:00:00 2001
From: "Christopher A. Williamson" <contact@chrisaw.io>
Date: Fri, 20 Feb 2026 14:08:15 +0000
Subject: [PATCH 3/3] Add VNDK v30 to extra VNDK versions

Include VNDK v30 prebuilts in the GSI image so devices shipping with
VNDK 30 vendor images can find the matching libraries at runtime.
---
 base.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/base.mk b/base.mk
index 37f6d38..457d03b 100644
--- a/base.mk
+++ b/base.mk
@@ -262,4 +262,4 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/window_extensions.mk)
 PRODUCT_PRODUCT_PROPERTIES += \
     persist.settings.large_screen_opt.enabled=true
 
-PRODUCT_EXTRA_VNDK_VERSIONS += 28 29
+PRODUCT_EXTRA_VNDK_VERSIONS += 28 29 30
-- 
2.52.0

